pkg/testers/node/node.go

```diff
@@ -242,6 +244,7 @@ func (t *Tester) constructArgs() []string {
 		"IMAGE_CONFIG_DIR=" + t.ImageConfigDir,
 		"SSH_USER=" + t.sshUser,
 		"SSH_KEY=" + t.privateKey,
+		"RUNTIME_CONFIG=" + t.RuntimeConfig,
```

Comment:
does it work correctly if it empty?

Reply:
Thanks for catching it out. This needs to be checked explicitly as it's been done for kubetest https://github.com/kubernetes/test-infra/blob/8c944aa6aa17c1efd224629b250e9bd52eeb7132/kubetest/e2e.go#L625-L627